Packs/Jira/Scripts/JiraCreateIssueExample/JiraCreateIssueExample.py

```diff
@@ -1,87 +1,109 @@
 
-def get_known_args_from_input(input: ItemsView) -> Dict[str, Any]:
+
+def add_custom_fields(args: Dict[str, Any], custom_fields: List[Dict[str, Any]]) -> Dict[str, Any]:
     """
-    Creates a dictionary of known arguments passed into script.
+    Method to generate the payload representing the Jira issue custom fields and add it to the script arguments.
+
     Args:
-        - `input` (`dict_items[Tuple[str, Any]]`): A view (dict_items) of the command arguments
+        - `custom_fields` (`List[Dict[str, Any]]`): A list of custom fields
     Returns:
-        - `Dict[str, Any]` representing the script arguments
+        - A dictionary of custom fields
     """
 
-    return {key: value for key, value in input if key in DEFAULT_ARGS}
+    if custom_fields:
+        args["issueJson"] = {}
+        args["issueJson"]["fields"] = {k: v for custom_field in custom_fields for k, v in custom_field.items()}
 
+    return args
 
-def get_additional_args_from_input(input: ItemsView) -> Dict[str, Any]:
+
+def rm_custom_field_from_args(args: Dict[str, Any]) -> Dict[str, Any]:
     """
-    Creates a dictionary of unknown arguments passed into script.
-    Args:
-        - `input` (`dict_items[Tuple[str, Any]]`): A view (dict_items) of the command arguments
-    Returns:
-        - `Dict[str, Any]` representing the script arguments
+    Method to remove the `customFields` dict from the command arguments.
+    jira-create-issue doesn't include `customFields` arg so we need to remove it and replace it with `issueJson`.
     """
 
-    return {key: value for key, value in input if key not in DEFAULT_ARGS}
+    del args["customFields"]
+
+    return args
 
 
 def main():  # pragma: no cover
     try:
 
-        input = demisto.args().items()
+        args = demisto.args()
+
+        demisto.debug(f"Arguments provided: \n{args}")
 
-        known_args = get_known_args_from_input(input)
-        additional_args = get_additional_args_from_input(input)
+        if "dueDate" in args:
+            validate_date_field(args.get("dueDate"))
 
-        # merge known and unknown into one
-        args = add_additional_args(known=known_args, additional=additional_args)
+        if "customFields" in args:
+            demisto.debug("Found customFields arguments. Attempting to parse them...")
+            custom_fields = parse_custom_fields(argToList(args.get("customFields")))
+            demisto.debug(f"Custom fields parsed: {custom_fields}. Removing 'customFields' argument...")
+            args = rm_custom_field_from_args(args)
+            demisto.debug("'customFields' removed. Adding custom field payload to the rest of the command arguments...")
+            propagated_args = add_custom_fields(args, custom_fields)
+            demisto.debug("Custom fields added to command arguments")
 
-        create_issue_result = demisto.executeCommand("jira-create-issue", args)
+        demisto.debug(f"Executing {INTEGRATION_COMMAND} with arguments: \n{propagated_args}")
```

Comment:
It's not necessary. I found it convenient to list it as a constant on top of the file so whoever reviews the code will know what underlying command it's using and als because it's used in two places in the code (`demisto.debug` and the `execute_command` call).